File: src/main/java/fr/tikione/c2e/model/web/TocCategory.java
package fr.tikione.c2e.model.web;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.List;

/**
 * TOC category that contains {@link TocItem} elements.
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class TocCategory {
    
    private String title;
    private List<TocItem> items = new ArrayList<>();
}

File: src/main/kotlin/fr/tikione/c2e/model/web/TocCategory.kt
package fr.tikione.c2e.model.web

import java.util.*

/**
 * TOC category that contains [TocItem] elements.
 */
class TocCategory {

    var title: String? = null
    var items = ArrayList<TocItem>()

    constructor() {
    }

    constructor(title: String?, items: ArrayList<TocItem>) {
        this.title = title
        this.items = items
    }
}
